src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -248,6 +254,8 @@ export const useTradeExecution = (
 
       switch (stepSellAssetChainNamespace) {
         case CHAIN_NAMESPACE.Evm: {
+          if (isExecutableTradeQuote(tradeQuote)) throw new Error('Unable to execute trade')
```

Comment:
also this and the cosmos sdk condition are inverted and wrong

Reply:
Consolidated in 32e54ca544 - note not at the very very top of the function so we ensure all event handlers are still present